libraries/fc/include/fc/static_variant.hpp

```diff
@@ -14,587 +14,86 @@
 #include <typeinfo>
 #include <fc/exception/exception.hpp>
 #include <boost/core/typeinfo.hpp>
+#include <variant>
 
 namespace fc {
 
-// Implementation details, the user should not import this:
-namespace impl {
-
-template<typename R, int N, typename... Ts>
-struct storage_ops;
-
-template<typename X, typename... Ts>
-struct position;
-
-template<int Pos, typename... Ts>
-struct type_at;
-
-template<typename... Ts>
-struct type_info;
-
-template< typename Visitor, typename ...Ts>
-struct const_result_type_info;
-
-template< typename Visitor, typename ...Ts>
-struct result_type_info;
-
-template< typename Visitor, typename ...Ts>
-struct rvalue_ref_result_type_info;
+// This will go away.
```

Comment:
When?

Reply:
The comment itself actually just needs to go away.

Reply:
The goal is to remove `fc::static_variant`, right? If this needs to be kept then I think it should be move to `raw.hpp`.

Reply:
Yes that was the goal but there is some custom functionality that was needed (`get_index`, `from_index`) that `std::variant` doesn't provide. So I don't know if this file is going away unless I can move all that functionality into raw.hpp or maybe there is a better place that I don't know about?